### PR TITLE
YOUR_USERNAME adds hometown of YOUR_HOMETOWN

### PR DESCRIPTION
**Pull request by githubteacher**
This PR is to add Orlando to the list of cities in the class repo.

Some questions I want answered are:
- Does anyone come to Central Florida looking for a particular type of food? 
- Is Central Florida synonymous with theme parks? Or are folks interested in natural attractions, as well?

### CODE REVIEW

**Comment by RobMaskell** [comment]
> Does anyone come to Central Florida looking for a particular type of food?

@githubteacher Italian food is always popular

**Comment by tallamraghava** [comment]
Interested to visit Clear water Beaches. Is that Good place to spend a full day ?

File: YOUR_USERNAME-YOUR_TOWN.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Orlando, FL
+
+### Restaurants
+- Italian: Italy Pavilion in Epcot has some really good authentic Italian food.
+
+### Tourist Attractions
+
+### Local Attractions
+- Forever Florida, Ecopark
+- Beaches are only 1-2 hours away. Clearwater is beautiful! Siesta Key is amazing!
```
Comment (JakeRuss) [inline YOUR_USERNAME-YOUR_TOWN.md]: +1 for Siesta Key

**Comment by githubteacher** [comment]
@RobMaskell great suggestion! I added it in b3a5be3
@tallamraghava added some beach suggestions in 34fe532

**Pull request merged**